### ISSUE TITLE
hydrogen output doesn't auto wrap

### ISSUE THREAD

**Issue by Jiawei-Navican** [nteract/hydrogen#1569]
**Description:**
The hydrogen output doesn't auto wrap long strings in atom block decoration or output panel.
<img width="800" alt="Screen Shot 2019-03-13 at 11 14 19 AM" src="https://user-images.githubusercontent.com/42557646/54313271-90eb4d80-459e-11e9-86d7-c8d7a1b5bb8d.png">
<img width="801" alt="Screen Shot 2019-03-13 at 2 46 09 PM" src="https://user-images.githubusercontent.com/42557646/54313358-cbed8100-459e-11e9-9b06-48e4cbfedba9.png">

**Versions:**

Which OS and which version of Hydrogen and Atom are you running?
MacOS High Sierra 10.13.6

Atom    : 1.35.1
Electron: 2.0.18
Chrome  : 61.0.3163.100
Node    : 8.9.3

**Logs:**
No error logs and output of the developer tools after following instructions on [Debugging Guide](https://nteract.gitbooks.io/hydrogen/docs/Troubleshooting.html).

**atom `config.json`:**
I am using atom both on windows and macOS, the `config.json` is synced between these two machine using `sync-settings`. So you can find windows pathes in the json file( I don't think it's related to this issue).
```
"*":
  Hydrogen:
    autoScroll: false
    debug: true
    outputAreaDock: true
  "atom-beautify":
    executables:
      php:
        path: "D:\\MAMP\\bin\\php\\php7.1.5\\php.exe"
    python:
      beautify_on_save: true
      formatter: "yapf"
      ignore: [
        "E24"
        "E731"
      ]
  "atom-ide-ui":
    "atom-ide-diagnostics-ui":
      showDiagnosticTraces: true
    "debugger-exceptionBreakpointsCollapsed": true
  "autocomplete-python": {}
  core:
    disabledPackages: [
      "markdown-preview"
      "atom-ide-debugger-python"
      "linter"
      "linter-ui-default"
      "language-python"
    ]
    projectHome: "C:\\Users\\jiawku\\git"
    telemetryConsent: "no"
    themes: [
      "atom-material-ui"
      "one-dark-syntax"
    ]
    uriHandlerRegistration: "always"
    useTreeSitterParsers: false
  editor:
    fontSize: 16
    preferredLineLength: 110
    softWrap: true
    softWrapAtPreferredLineLength: true
    tabLength: 4
  "exception-reporting":
    userId: "ac6b7838-f119-49a2-8cea-41491d3f74cd"
  "ide-python":
    pylsPlugins:
      autopep8:
        enabled: false
      pydocstyle:
        enabled: true
      yapf:
        enabled: true
  kite:
    showWelcomeNotificationOnStartup: false
  "linter-flake8":
    ignoreErrorCodes: [
      "E501"
      "E731"
    ]
  "linter-ui-default":
    panelHeight: 300
  "markdown-preview-enhanced":
    codeBlockTheme: "default.css"
    pandocPath: "C:\\Users\\jiawku\\AppData\\Local\\Pandoc\\pandoc.exe"
    phantomPath: "D:\\phantomjs-2.1.1-windows\\bin\\phantomjs.exe"
  "sync-settings":
    _lastBackupHash: "5935aee06c8505149f9dd0bc5ea08794ede3a9d9"
    gistId: "260fa204be9fe4fa1720f9773840c508"
    personalAccessToken: "91a5c955cbab05ee0a277217dd5b9b127e6a4ffc"
  welcome:
    showOnStartup: false
".breakpoints.hydrogen":
  editor:
    preferredLineLength: 80
    softWrap: true
    softWrapAtPreferredLineLength: true
```

**atom `style.less`:**
```
// style the background color of the tree view
.tree-view {
  // background-color: whitesmoke;
}

// style the background and foreground colors on the atom-text-editor-element itself
atom-text-editor {
  // color: white;
  // background-color: hsl(180, 24%, 12%);
}

// style UI elements inside atom-text-editor
atom-text-editor .cursor {
  // border-color: red;
}
```

My UI theme is `Atom Material` and Syntax Theme is `One Dark`.

Thank you, guys.

**Comment by BenRussert** [comment]
This _appears_ to be a css bug. It could be in the `./lib/result-view` component or in our `styles.less` file. 

I'd love it if someone took a crack at this one and reported back any findings. You might even be able to play around with your atom stylesheet/devtools and get something that works, then make a PR or comment back here.

**Comment by aviatesk** [comment]
I tweaked some styles and found that changing the style below will wrap the results both in inline cell blocks & the output pane:
> styles/hydrogen.less
```less
...
.hydrogen {
  code {
    white-space: pre-line;  // Changed from `pre`
}
...
```

And the results will be like:
![image](https://user-images.githubusercontent.com/40514306/56084926-d6d55300-5e75-11e9-8508-ad356c8ff45b.png)

I think it would be fancy if we can make a result wrapped or not through the config setting.
How about the idea, @BenRussert ? I'm welcome to implement that.

**Comment by BenRussert** [comment]
Great catch @aviatesk! Hope you send a PR.

>  result wrapped or not through the config setting.

I think this is a fix that most people want, so a config probably isn't needed. If people want they can always set their personal atom stylesheet.

**Comment by aviatesk** [comment]
@BenRussert 
> I think this is a fix that most people want, so a config probably isn't needed. If people want they can always set their personal atom stylesheet.

Okay I understood, and welcome to make a PR !

**Comment by BenRussert** [comment]
Closed by #1594

**Comment by wadethestealth** [comment]
@BenRussert @aviatesk It appears this issue was never a bug at all. Your default behavior was to use scrolling, but apparently in their images they don't see the scroll bar. I am currently implementing a configuration for soft wrapping, but there was never actually an issue.
